docs/flags.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Tiltfile Flags and Args
+layout: docs
+---
+
+
+Tools offer flags and args to support the wide range of users needs and preferences, and dev workflows are no different. Devs on your team will want options in how they use Tilt; some are long-standing personal preferences and others will change in the middle of working on a Pull Request. With Tilt's user configuration system, you can improve a Tiltfile that encodes one standard setup to power the breadth of workflows your team needs:
```

Comment:
This sentence, and this doc, feels weird in that it focuses on Laura in a way that our other docs don't. It might be unnecessarily exclusionary to readers.

Reply:
I agree it's geared towards that user, and intentionally. I think exclusionary has a negative connotation. Do you think someone else would not read it? I figured they'd read it understanding the perspective. I'm not sure how to write it well for another user.